src/antsibull_docs/data/docsite/list_of_collections.rst.j2

```diff
@@ -1,3 +1,9 @@
+{#
+  Copyright (c) Ansible Project
+  GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+  SPDX-License-Identifier: GPL-3.0-or-later
+#}
+
```

Comment:
This (and some of the other jinja2 files) use Jinja2 comments that don't show up in the rendered file. Is that on purpose?

Reply:
Yes, it's intentional. The generated RST files mainly contain information from the collection the docs are generated of.

Reply:
Wouldn't the generated RST files still be considered GPL, as the template is?

Reply:
I guess that's a question that will spark very interesting legal discussions among legal experts ;-) I have no idea. I can think of reasons for both "yes" and "no", so here I would avoid trying to take a side on this question by avoiding to insert copyright or license statements into the generated RST.

Reply:
> I would avoid trying to take a side on this question by avoiding to insert copyright or license statements into the generated RST.

I guess. We might want to re-license the files under a very permissive license such as [MIT-0](https://spdx.org/licenses/MIT-0.html) (a MIT license variant that doesn't require any attribution), but that's another conversation.

You can resolve this thread.

Reply:
That also might require quite some effort, since these templates are based on templates in ansible/ansible, which have been edited by quite a few of folks over some years. Figuring out who was involved and asking all of them will be some effort. Doing that would be a good idea though IMO, but I don't want to do that anytime soon :)